wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,73 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I restricted?
+A user may get restricted if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What is considered cheating?
```

Comment:
The [rules](/wiki/Rules) say that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player is considered cheating.